.github/workflows/bench.yml

```diff
@@ -1,9 +1,11 @@
 name: Benchmarks
 on:
   pull_request:
-  push:
```

Comment:
I suppose we still want to run the local benchmark workflow on master builds

Reply:
I think we can run the local ones on every PR too, shouldn't be a big issue?

Reply:
of course, I was saying as a well as the master branch